lib/cli/src/commands/app/create.rs

```diff
@@ -74,6 +90,320 @@ pub struct CmdAppCreate {
     /// Name of the package to use.
     #[clap(long, short = 'p')]
     pub package: Option<String>,
+
+    /// Whether or not to search (and use) a local manifest.
+    #[clap(long)]
+    pub use_local_manifest: bool,
+
+    /// Name to use when creating a new package from a template.
+    #[clap(long)]
+    pub new_package_name: Option<String>,
```

Comment:
This should be `--template-package-name` (is longer, but more exhaustive than `new`

Reply:
Let's delete this `new_package_name` and use `package` instead (this flow will only be triggered if `--template` is provided or the user specifies the template